docs/reference-use-cases.md

```diff
@@ -0,0 +1,215 @@
+
+- As an operator of a management cluster, I want to know what versions of kublet, control plane, OS, etc all of the associated workload clusters are running, so that I can plan upgrades to the management cluster that will not break anyone’s ability to manage their workload clusters.
+
+### Removing Cluster API
+- As an operator of a management cluster, given that I have a management cluster that I have used to deploy several workload clusters, I want to remove the Cluster/Machine objects representing one workload cluster from my management cluster without deprovisioning workload cluster.
+
+- As an operator of a management cluster, given that I have a management cluster that I have used to deploy several workload clusters, I want to uninstall Cluster API from my management cluster without deprovisioning my workload clusters.
+
+- As an operator of a management cluster, given that I have a management cluster, I want to use it to manage workload clusters that were created by a different management cluster.
+
+### Cross-cluster Metrics
+- As an operator of a management cluster, I want to query my resource allocation on an infrastructure.  For example, in an on-prem case, I do not have an infinite capacity cloud, so  I need to be able to determine my reservation before deploying a workload cluster.
+
+### Specific Architecture Approaches
+- As an operator of a management cluster, given that I give operators of workload clusters access to my management cluster, they can launch new workload clusters with control planes that run in the management cluster while the nodes of those workload clusters run elsewhere.
+
+- As a multi-cluster operator, I would like to provide an EKS-like experience. Where the workload control plane nodes are joined to the management cluster and the control plane config isn’t exposed to the consumer of the workload cluster. This enables me as an operator to manage the control plane nodes for all clusters using tooling like prometheus and fluentd. I can also control the configuration of the workload control plane in accordance with business policy.
+
+### Multitenancy Management
+- As an operator of a management cluster, I want to control which users of management cluster can deploy new workload clusters, how many clusters they can deploy, and how many nodes/resources those clusters can use.
+
+- As an operator of a management cluster, I want to ensure that only the user who creates a new workload cluster (and some specific other users) can manage and access the workload cluster.
+
+- As an operator of a management cluster, I want the user who creates a new workload cluster to be able to give permission to other users to manage that cluster.
+
+- As an operator of a management cluster, I want to configure whether operators of workload clusters are allowed to open interactive shells onto those clusters machines.
+
+## Multi-cluster/Multi-provider
```

Comment:
How about adding a `Multi-vendor` section, there are now many different k8s distributions, like IBM Cloud Private, OpenShift, K3S etc, and end user may have requirement to install those distributions but not only k8s native cluster.

Reply:
I saw we do have mentioned `vendor's cluster` at https://github.com/kubernetes-sigs/cluster-api/blob/55cdc685cd0319a07b875d793d94ec54e7441d95/docs/reference-use-cases.md#creating-clusters , this is great!